### PR TITLE
Fix "debug" in new version node.js

### PR DESCRIPTION
**Pull request by hermanho**
## Proposed changes

[//]: # (Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.)

Bug fixing 
https://github.com/webdriverio/webdriverio/issues/1640#issuecomment-292419434
https://github.com/webdriverio/webdriverio/issues/2194

References:
https://nodejs.org/en/docs/guides/debugging-getting-started/#legacy-debugger
https://code.visualstudio.com/docs/nodejs/nodejs-debugging#_supported-nodelike-runtimes

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [ ] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

File: lib/launcher.js [hunk 1]
```diff
@@ -303,6 +303,17 @@ class Launcher {
...
+        
+        if (debug) {
+            for (let i in process.execArgv) {
+                if (process.execArgv[i].startsWith("--debug-brk")) {
+                    debugArgs.push('--inspect-brk');
+                } else if (process.execArgv[i].startsWith("--inspect=")) {
+                    debugArgs.push('--inspect=' + (process.debugPort + processNumber));
+                }
+            }
+            console.debug(debugArgs);
```
Comment (christian-bromann) [inline lib/launcher.js]: Is this console.log necessary?

Comment (hermanho) [inline lib/launcher.js]: It can be removed, debugging purpose.

**Comment by mltsy** [comment]
Are we just waiting for this console.log to be removed in order merge this so we can use debug with newer node versions?  Shall I fork the branch and remove it in a new PR?

**Comment by christian-bromann** [comment]
> Shall I fork the branch and remove it in a new PR?

Yes, that would be great to move forward here. Thanks @mltsy

**Comment by mltsy** [comment]
Done!  Please see my "Further comments" section on the new PR @christian-bromann

**Comment by christian-bromann** [comment]
Closing in favor of #2350

**Pull request closed without merging**